src/opnsense/service/templates/OPNsense/IDS/suricata.yaml

```diff
@@ -70,6 +70,29 @@ stats:
 
 # Configure the type of alert (and other) logging you would like.
 outputs:
+{% if helpers.exists('OPNsense.IDS.general.unix_sock') and OPNsense.IDS.general.unix_sock == '1' %}
+
+  # Extensible Event Format (nicknamed EVE) to UNIX-socket
+  - eve-log:
+      enabled: yes
+      filetype: unix_stream
+      filename: /tmp/suricata-stats.sock
+      types:
```

Comment:
probably best to align output to the current syslog/local versions, e.g. implement payload options when `LogPayload` is enabled.

Reply:
I am sorry, this I did not comprehend.

Reply:
eve output is already defined twice (syslog and standard file), best make sure that the options about the log format are shared among all of them, in which case the log payload setting is missing in this section (just search for LogPayload in  the template and you see what I mean).

Reply:
I think I understood. I have updated the section to include payload options.

Reply:
thanks, looks good